iowa-a/bedrock-prod/canary-deploy.yaml

```diff
@@ -195,7 +195,7 @@ spec:
               key: twitter-consumer-secret
               name: bedrock-prod-secrets
         - name: WEB_CONCURRENCY
-          value: "6"
+          value: "20"
```

Comment:
From checking the transaction in New Relic, it seems like bedrock spends most of its time in python code vs db or network calls. It's hard to understand why we'd need so many.

No objections to canarying with this number, but I think if it works we should determine how to set an optimal number of workers per pod.

Reply:
Yeah, I was trying to choose a number so high there'd be no question it was high enough.  20 should be more than enough, and then backing down to a more reasonable number is easier.